internal/secant/attest.go

```diff
@@ -66,6 +67,16 @@ func Attest(ctx context.Context, statements []*types.Statement, sv types.Cosigne
 	ropts := []ociremote.Option{ociremote.WithRemoteOptions(ropt...)}
 	se := ociremote.SignedUnknown(digest, ropts...)
 
+	statements, err := newStatements(statements, se)
```

Comment:
Can we make this opt-in?  Maybe we should even have an enum: REPLACE | APPEND | SKIPSAME?

Reply:
I'm imagining...

APPEND just always adds every statement.
REPLACE replaces by predicateType.
SKIPSAME replaces by predicateType but only if the payloadHash is different?

Reply:
yeah